content/tutorials/manage-the-rippled-server/configure-peering/enable-link-compression.md

```diff
@@ -0,0 +1,52 @@
+---
+html: enable-link-compression.html
+funnel: Docs
+doc_type: Tutorials
+category: Manage the rippled Server
+subcategory: Configure Peering
+blurb: Save bandwidth by compressing peer-to-peer communications.
+---
+# Enable Link Compression
+
+The `rippled` server can save bandwidth by compressing its [peer-to-peer communications](peer-protocol.html), at a cost of greater CPU usage. If you enable link compression, the server automatically compresses communications with peer servers that also have link compression enabled. The server uses uncompressed communications with peers who do not have have link compression enabled or do not support link compression. [New in: rippled 1.6.0][]
+
```

Comment:
Was wondering how much CPU usage would be saved by this. Makes sense to compress bandwidth if you knew this setting is the norm for most other hosts.

Anyway to recast the last sentence? There's. also a repeated "have" in the sentence:

The server uses uncompressed communications with peers who do not **have have** link compression enabled or do not support link compression.

Something like this, maybe?

Peer-to-peer link compression works only with peer servers that support link compression and have link compression enabled. In the absence of either, your server defaults to uncompressed communications with that peer.

Reply:
Was thinking about it and I decided the best way to rephrase the sentence was to remove it.

It's implied by the previous sentence and awkwardly long pretty much no matter what, so I think in reality it's more likely to confuse people than it is to help.